cmd/wasm-run/main.go

```diff
@@ -95,6 +97,9 @@ func run(w io.Writer, fname string, verify bool) {
 }
 
 func importer(name string) (*wasm.Module, error) {
+	if name == "env" {
```

Comment:
Is the 'env' import reserved?

Could someone legitimately create a wasm file named 'env' and we break them?

Reply:
I've not found it in the WASM spec...it looks to me like its just a 'convention' used internally by emscripten, and possible adopted by LLVM in their code (I'll research more).

That is why it has to be optional

The above was a temporary for the POC

Reply:
Ah gotcha :) plz move it to a flag or something before we merge.

Reply:
Of course, I'm sorry to be clear this pull request was meant to start the conversation not be syntactically correct...I'll work on a better implementation now that we have nailed down a few things

Reply:
Ahhhh, my bad, I didnt realise :O

In that case LGTM.